### PR TITLE
Fix/pic state

### PR DESCRIPTION
**Pull request by krampstudio**
Enable to get/set the state of a pic using the standard methods.

### CODE REVIEW

File: views/js/qtiItem/helper/util.js [hunk 1]
```diff
@@ -11,6 +33,92 @@ define(['lodash'], function(_){
...
         },
+
+        /**
+         * Generates an id for a Qti element (the generation is different from identifier)
+         * @param {Object} item - the element related item
+         * @param {String} prefix - identifier prefix
+         * @returns {String} the identifier
+         * @throws {TypeError} if there is no item
+         */
+        buildId : function buildSerial(item, prefix){
```
Comment (ssipasseuth) [inline views/js/qtiItem/helper/util.js]: You mean buildSerial -> buildId ?

**Comment by ssipasseuth** [comment]
Ok, it looks good to me. Would only need some testing to make sure it works.

**Comment by jsconan** [comment]
The test #16 `Provider PIC: PIC state` never ends under Firefox, but it does under Chrome.
image

There is an error in the console: `TypeError: state.pic is undefined #(test.js:506)`

**Comment by krampstudio** [comment]
ready for a new review;

**Pull request merged**